### DISCUSSION TITLE
Use MSSQL (Microsoft SQL Server) as DB

### DISCUSSION THREAD

**Discussion by kautsaridris** [NginxProxyManager/nginx-proxy-manager#1711] [Q&A]
I want to know if may to make connection to sql server (microsoft) using NPM ?

**Comment by chaptergy** [comment]
Please, don't just tag people to try to get noticed. 

About your question: No MSSQL is not supported. Currently only MySQL / MariaDB and SQLite is supported, and going forward (in v3) MySQL / MariaDB support will be dropped in favor of the much less error prone SQLite (since it runs in the same container).
